packages/plugin/src/Services/DiagnosticsService.php

```diff
@@ -572,4 +598,32 @@ private function getSummary(): InstallSummary
 
         return $summary;
     }
+
+    private function getIntegrationCount(): array
+    {
+        $integrations = (new Query())
+            ->select(['forms_integrations.id', 'forms_integrations.integrationId', 'forms_integrations.formId', 'integrations.class'])
+            ->from(FormIntegrationRecord::TABLE.' forms_integrations')
+            ->innerJoin(IntegrationRecord::TABLE.' integrations', 'integrations.id = forms_integrations.integrationId')
+            ->where(['forms_integrations.enabled' => true])
+            ->all()
+        ;
+
+        $integrationsByForm = [];
+
+        foreach ($integrations as $integration) {
+            $id = $integration['integrationId'];
+            if (!isset($integrationsByForm[$id])) {
+                $integrationsByForm[$id] = [
+                    'name' => $this->integrationTypeProvider->getTypeDefinition($integration['class'])->name,
+                    'version' => $this->integrationTypeProvider->getTypeDefinition($integration['class'])->version,
+                    'count' => 0,
+                ];
```

Comment:
Since we're using that huge path of a parameter more than once, it makes for cleaner code to extract it into a variable.

```suggestion
                $type = $this->integrationTypeProvider->getTypeDefinition($integration['class']);
                $integrationsByForm[$id] = [
                    'name' => $type->name,
                    'version' => $type->version,
                    'count' => 0,
                ];
```

And then, to prevent the `version` from being `null`, we can default it to an empty string.


```
$type = $this->integrationTypeProvider->getTypeDefinition($integration['class']);
$integrationsByForm[$id] = [
    'name' => $type->name,
    'version' => $type->version ?? '',
    'count' => 0,
];
```